Hunting Queries/LAQueryLogs/NewClientUsed.yaml

```diff
@@ -0,0 +1,28 @@
+id: 1dd98313-f43f-4d8b-9870-5a1dfb2cf93f
+name: New client running queries
+description: |
+  'This hunting query looks for clients running queries that have not previously been seen running queries.'
+requiredDataConnectors:
+  - connectorId: CustomConnector
```

Comment:
Lets use connectorId: "AzureMonitor(Query Audit)" for now, it may change slightly but should be under Azure Monitor.